node/src/edgeAuth.js

```diff
@@ -36,74 +36,78 @@ program
   .option('-m, --room <roomId>', '[STREAMING] Token is limited to the given room')
   .option('-n, --roomAlias <roomAlias>', '[STREAMING] Token is limited to the given room alias')
   .option('-t, --tag <tag>', '[STREAMING] Token is limited to the given origin stream tag')
-  .option('-r, --applyTag <applyTag>', '[REPORTING] Apply tag to the new stream');
+  .option('-r, --applyTag <applyTag...>', '[REPORTING] Apply tag to the new stream');
 
 program.parse(process.argv);
 
+console.log(program);
+
+const options = program.opts();
+console.log(options)
```

Comment:
The console output is an API contract. It makes little sense that the program and options are printed back in production use cases.